aidial_adapter_bedrock/bedrock.py

```diff
@@ -36,6 +36,26 @@ async def acreate(cls, aws_client_config: AWSClientConfig) -> "Bedrock":
         )
         return cls(client)
 
+    @classmethod
+    def create(cls, aws_client_config: AWSClientConfig) -> "Bedrock":
```

Comment:
The `create` method is used only once - in tests. 

Why not use `acreate` there instead and remove this method?

Reply:
Removed